src/probability_theory/independence.lean

```diff
@@ -121,7 +121,7 @@ Indep (λ x, measurable_space.comap (f x) (m x)) μ
 independent. For a function `f` with codomain having measurable space structure `m`, the generated
 measurable space structure is `measurable_space.comap f m`. -/
 def indep_fun {α β γ} [measurable_space α] (mβ : measurable_space β) (mγ : measurable_space γ)
```

Comment:
What about putting `mβ` and `mγ` as typeclass assumptions? In most cases, the functions will be real-valued or ennreal-valued, and we will use the standard sigma-algebra in the target, so we could as well let it be inferred by the system.

Reply:
I have a lot of thoughts on independence, but should those be handled in a separate PR?

Reply:
Okay, that can be done in a separate PR